pkg/controller/controller.go

```diff
@@ -95,6 +96,13 @@ func (c AppGwIngressController) Process(event QueuedEvent) error {
 
 		glog.V(5).Infof("AzureIngressProhibitedTargets: %+v", strings.Join(prohibitedTargets, ","))
 	}
+	{
```

Comment:
This is great!
Let's wrap this with a feature flag so we don't surprise our customers with unnecessary log lines.
I already introduced the following environment variable:
https://github.com/Azure/application-gateway-kubernetes-ingress/blob/85f945ddf5b2a5f0f9a785b389c989b15033a5a6/pkg/environment/environment.go#L41

Here is an example of a feature flag:
https://github.com/Azure/application-gateway-kubernetes-ingress/blob/85f945ddf5b2a5f0f9a785b389c989b15033a5a6/pkg/k8scontext/context.go#L272

So here on this line we could swap the `{` with `if envVariables.EnableIstioIntegration == "true" {`